spec/models/movie_spec.rb

```diff
@@ -1,7 +1,37 @@
 require "rails_helper"
 
 RSpec.describe Movie, type: :model do
+  before :each do
+    @us = Studio.create!(name: 'Universal Studio', location: 'Hollywood')
+
+      @marry = @us.movies.create!(title: 'Marry Me', creation_year: '2022', genre: 'Romance')
+  
+      @jenni = Actor.create!(name: 'Jennifer Lopez', age: 54)
+      @owen = Actor.create!(name: 'Owen Wilson', age: 55)
+      @sarah = Actor.create!(name: 'Sarah Silverman', age: 52)
+      @chloe = Actor.create!(name: 'Chloe Coleman', age: 14)
+  
+      MovieActor.create!(movie: @marry, actor: @jenni)
+      MovieActor.create!(movie: @marry, actor: @owen)
+      MovieActor.create!(movie: @marry, actor: @sarah)
+      MovieActor.create!(movie: @marry, actor: @chloe)
+  end
+  
   describe "relationships" do
     it {should belong_to :studio}
+    it {should have_many :movie_actors}
+    it {should have_many(:actors).through(:movie_actors)}
+  end
+
+  describe 'actors_list' do
+    it 'lists the actors from youngest to oldest' do
+      expect(@marry.actors_list).to eq([@chloe, @sarah, @jenni, @owen])
+    end
+  end
+
+  describe 'ave_actors_age' do
+    it 'averages actors age for the movie' do
+      expect(@marry.ave_actors_age).to eq(44)
+    end
   end
```

Comment:
Great job testing your instance methods. Like I mentioned before I personally wouldn't have used `ceil` in my logic inside of my method and instead would have used it in my view and in my tests. My tests would have looked something like this:

```ruby
expect(@marry.ave_actors_age.ceil).to eq(44)
```